src/main/java/com/schematical/chaoscraft/entities/EntityFitnessScoreEvent.java

```diff
@@ -27,6 +27,6 @@ public float getAdjustedScore(){
         return Math.round(score * multiplier);
     }
     public String toString(){
-        return worldEvent.toString() + " = " + getAdjustedScore() + "(" + score + "*" + multiplier + ")";
```

Comment:
This fix isnt going to cut it. Unless we have some type of ScoreEvent detail page I still want that data.

Reply:
This is the fix I want: https://github.com/schematical/chaoscraft-mod/commit/ca7c46f444281b7d75400a6403edc6fba15a9a23
Pull out this line from your pull request and we are good to go